homeassistant/components/apsystemsapi_local/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""The config_flow for APsystems local API integration."""
+
+from aiohttp import client_exceptions
+from APsystemsEZ1 import APsystemsEZ1M
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_IP_ADDRESS, CONF_NAME
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN, LOGGER, UPDATE_INTERVAL
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_IP_ADDRESS): str,
+        vol.Required(CONF_NAME): str,
+        vol.Optional(UPDATE_INTERVAL, default=15): int,
```

Comment:
Don't expose the update interval to the user. That should be set by integration author.

Reply:
Yea. The integration is also available through HACS and some users wanted to update the data every 5 seconds, but that's not needed for others. They want to adjust the power output (will be added in another PR as it's a different platform) of the inverter dynamically so that no energy gets "put" into the grid.